src/main.ts

```diff
@@ -38,6 +47,6 @@ async function bootstrap() {
   app.use(passport.initialize());
   app.use(passport.session());
   app.use(cookieParser());
-  await app.listen(3000);
+  await app.listen(8080);
```

Comment:
Why was this changed? The docker port is 3000

Reply:
Please change this